xds/internal/balancer/edsbalancer/configbuilder.go

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package edsbalancer
+
+import (
+	internalserviceconfig "google.golang.org/grpc/internal/serviceconfig"
+	"google.golang.org/grpc/resolver"
+	"google.golang.org/grpc/xds/internal/balancer/clusterresolver/balancerconfigbuilder"
+	xdsclient "google.golang.org/grpc/xds/internal/client"
+)
+
+const million = 1000000
+
+func buildPriorityConfigMarshalled(edsResp xdsclient.EndpointsUpdate, c *EDSConfig) ([]byte, []resolver.Address) {
```

Comment:
maybe s/Marshalled/JSON/ or simply skip the `Marshalled` part?
Also Marshaled has a single `l` I think

Reply:
Renamed to `xxxJSON`.